libraries/api/src/middleware/schema.js

```diff
@@ -90,8 +90,8 @@ const validateSchemas = ({ validator, absoluteSchemas, rootUrl, serviceName, ent
       if (!json) {
         return res.status(204).send();
```

Comment:
I know this isn't the normal case but if I specify an undefined `json` and pass a `responseCode` I feel like maybe this should reply with the response code I provide?

Reply:
Fair enough - I've added that.  I'll merge if green